juniper_official/vpn/evpn/evpn-check-mac-count-netconf.rule

```diff
@@ -0,0 +1,114 @@
+             * to the shorter field names used in the rules.
+             */
+            field mac-count {
+                sensor evpn-mac-count {
+                    path mac-count;
+                }
+                type integer;
+                description "This filed is for mac count";
+            }
+            field instance-name {
+                sensor evpn-mac-count {
+                    where "instance-name =~ /{{instance-name}}/";
+                    path instance-name;
+                }
+                type string;
+                description "This filed is for rtb name in mac table";
+            }
+            field threshold {
+                constant {
+                    value "{{static-threshold}}";
+                }
+                type integer;
+            }
+            field learn-vlan {
+                sensor evpn-mac-count {
+                    path learn-vlan;
+                }
+                type string;
+                description "VLAN ID";
+            }
+            /*
+             * Anomaly detection logic.
+             */
+            trigger mac-count {
+                synopsis "MAC count kpi";
+                description "Sets health based on evpn mac count";
+                frequency 1.5offset;
+                term is-mac-count-above-static-threshold {
+                    when {
+                        greater-than-or-equal-to "$mac-count" "$threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "MAC count ($mac-count) of instance $instance-name vlan $learn-vlan is above threshold($threshold)";
+                        }
+                    }
+                }
+                term mac-count-normal {
+                    then {
+                        status {
+                            color green;
+                            message "MAC count ($mac-count) of instance $instance-name vlan $learn-vlan is normal";
+                        }
+                    }
+                }
+            }
+           /*
+            * Variables with default values. Default values can be changed
+            * while deploying playbook instance.
+            */
+            variable static-threshold {
+                value 800;
+                description "Enter MAC limit threshold per VLAN";
+                type int;
+            }
+            variable instance-name {
+                value .*;
+                description "Enter instance-name to be monitored.";
+                type string;
+            }
+            rule-properties {
+                version 1;
+                contributor juniper;
+                supported-healthbot-version 4.0.0;
+                supported-devices {
+                    juniper {
+                         operating-system junosEvolved {
+                             products ACX {
```

Comment:
Please fill details of supported devices..PLease reach out to Vikram for details

Reply:
Updated